lib/matplotlib/tests/test_ticker.py

```diff
@@ -536,12 +536,66 @@ def test_formatstrformatter():
 
 
 @pytest.mark.parametrize('xmax, decimals, symbol, x, display_range, expected',
-                         percentformatter_test_cases)
```

Comment:
same comment as on other PR, maybe keep the testcase and id outside of the decorator and just refer to 'em?